lib/lighthouse/letters_generator/service_error.rb

```diff
@@ -16,8 +30,29 @@ def initialize(exception = nil)
           @status ||= exception['status']
           @title ||= exception['title']
           @message = exception['detail'] || exception['message']
+          @key ||= error_key
         end
       end
+
+      def errors
+        Array(
+          Common::Exceptions::SerializableError.new(
+            i18n_data.merge(title: @title, detail: @message, source: 'Lighthouse::LettersGenerator::Service')
+          )
+        )
+      end
+
+      private
+
+      def i18n_key
+        @key
+      end
+
+      def error_key
+        return ERROR_MAP[@status] if ERROR_MAP.key?(@status)
+
+        ERROR_MAP[:default]
```

Comment:
In Ruby, `nil` is falsey. Could this be simplified to:

`ERROR_MAP[@status] || ERROR_MAP[:default]`